doc/modules/array_api.rst

```diff
@@ -106,6 +106,7 @@ Metrics
 
 - :func:`sklearn.metrics.accuracy_score`
 - :func:`sklearn.metrics.zero_one_loss`
+- :func:`sklearn.metrics.mean_tweedie_deviance`
```

Comment:
```suggestion
- :func:`sklearn.metrics.accuracy_score`
- :func:`sklearn.metrics.mean_tweedie_deviance`
- :func:`sklearn.metrics.zero_one_loss`
```

(it looks like there's a tradition to keep those lists lexicographically ordered)